rbac/management/role/view.py

```diff
@@ -29,6 +29,8 @@
 from .model import Role
 from .serializer import RoleSerializer
 
+APP_WHITELIST = ['app', 'cost-management']
```

Comment:
Hm, trying to think if there'd be a more graceful way to handle the test allowance here. Having to put something specific in the app code for testing isn't ideal. Can we maybe make use of the .env file? Or would it be a huge headache to update the tests to just reference `cost-management`? 

It just might be confusing having this `app` allowance here, and although like you said it doesn't expose anything, it does allow you to create those orphaned `app` roles.

Reply:
We could update the testing to use 'cost-management' but that feels hacky rather than using a specific 'testing' application. Maybe something with the `.env` file would work. I'll look into that a bit.

Reply:
@wcmitchell - Sounds good! Likewise, coupling the app logic to tests also doesn't feel right, if we're able to avoid it.